lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -654,24 +654,34 @@ void
 IRBuilderAsmJs::BuildHeapBufferReload(uint32 offset)
 {
 
-    // ArrayBuffer (or WebAssembly.Memory in Wasm)
-    IR::RegOpnd * dstOpnd = BuildDstOpnd(AsmJsRegSlots::ArrayReg, TyVar);
-    IR::Opnd * srcOpnd = IR::IndirOpnd::New(BuildSrcOpnd(AsmJsRegSlots::ModuleMemReg, TyVar), (int32)Js::AsmJsModuleMemory::MemoryTableBeginOffset, TyVar, m_func);
-    IR::Instr * instr = IR::Instr::New(Js::OpCode::Ld_A, dstOpnd, srcOpnd, m_func);
-    AddInstr(instr, offset);
 #ifdef ENABLE_WASM
     if(m_func->GetJITFunctionBody()->IsWasmFunction())
     {
+        // WebAssembly.Memory
+        IR::RegOpnd * dstOpnd = BuildDstOpnd(AsmJsRegSlots::WasmMemoryReg, TyVar);
+        IR::Opnd * srcOpnd = IR::IndirOpnd::New(BuildSrcOpnd(AsmJsRegSlots::ModuleMemReg, TyVar), (int32)Js::AsmJsModuleMemory::MemoryTableBeginOffset, TyVar, m_func);
```

Comment:
Should use Js::WebAssemblyModule::GetMemoryOffset() here instead. 
It's not a big deal since it has to be the same for wasm and asm.js now, but might as well use the wasm version